test/integration/resolver.test.js

```diff
@@ -18,6 +19,8 @@ import {
 } from 'graphql';
 
 describe('resolver', function () {
+  beforeRemoveAllTables();
```

Comment:
Unfortunately no... The problem come from the test executed before. It creates tables and `sequelize.modelManager.models = [];` in the before made impossible to reset all tables.